cmd/stack_ls.go

```diff
@@ -26,21 +27,47 @@ import (
 	"github.com/pulumi/pulumi/pkg/backend/display"
 	"github.com/pulumi/pulumi/pkg/backend/httpstate"
 	"github.com/pulumi/pulumi/pkg/backend/state"
-	"github.com/pulumi/pulumi/pkg/tokens"
 	"github.com/pulumi/pulumi/pkg/util/cmdutil"
 	"github.com/pulumi/pulumi/pkg/workspace"
 )
 
 func newStackLsCmd() *cobra.Command {
-	var allStacks bool
 	var jsonOut bool
+
```

Comment:
NIT: could you remove the blank here so all the values are one after another?  I think that's more in line with other commands.